modules/node_modules/@ncigdc/components/QuickSearch/QuickSearch.js

```diff
@@ -66,7 +83,7 @@ compose(
       setState(s => ({ ...s, query, isLoading: true }));
       const timeOfCurrentRequest = new Date().getTime();
       timeOfMostRecentRequest = timeOfCurrentRequest;
-      const url = `${api}all?fields=${SEARCH_FIELDS.join(',')}&query=${window.encodeURIComponent(query)}&size=10`;
+      const url = `${api}all?fields=${SEARCH_FIELDS.join(',')}&query=${window.encodeURIComponent(query)}&size=5`;
```

Comment:
since your updating this line, you could switch it to use this function https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/utils/ajax/index.js#L28